connectors/src/lib/models.ts

```diff
@@ -0,0 +1,120 @@
+import {
+  type CreationOptional,
+  DataTypes,
+  type ForeignKey,
+  type InferAttributes,
+  type InferCreationAttributes,
+  Model,
+  Sequelize,
+} from "sequelize";
+
+import { type ConnectorProvider } from "../types/connector";
+
+const { CONNECTORS_DATABASE_URI } = process.env;
+
+export const sequelize_conn = new Sequelize(
+  CONNECTORS_DATABASE_URI as string,
```

Comment:
Might be better to have `if (! CONNECTORS_DATABASE_URI) {throw Something}` (`CONNECTORS_DATABASE_URI` will become a string after that point)

Reply:
done